clouddriver-google/src/main/java/com/netflix/spinnaker/clouddriver/google/config/GoogleCredentialsConfiguration.java

```diff
@@ -111,4 +116,13 @@ public void synchronize() {
       }
     };
   }
+
+  @Bean
+  @ConditionalOnMissingBean(
+      value = GoogleNamedAccountCredentials.class,
+      parameterizedContainer = CredentialsRepository.class)
+  public CredentialsRepository<GoogleNamedAccountCredentials> googleCredentialsRepository(
```

Comment:
I realize it's potentially painful, but I gotta ask...what's involved in adding a test for this?